src/handlers/comment/handlers/index.ts

```diff
@@ -42,7 +42,11 @@ export const issueClosedCallback = async (): Promise<void> => {
   const issue = (_payload as Payload).issue;
   try {
     const comment = await handleIssueClosed();
-    return await addCommentToIssue(comment!, issue!.number);
+    await addCommentToIssue(comment!, issue!.number);
+    await addCommentToIssue(
```

Comment:
It seems that these should be consolidated into a single comment. What is :45 commenting exactly?

Reply:
:45 is the returning comment about payment permit when the user closed issue. And if we consolidate them into one, it will be hard to impact review comment. Is that OK?
I mean, the payment permit result and "If you enjoy the experience ..." comment will be one comment.

Reply:
Do you have an example on your fork so I could see and compare?